purchase_receipt_progress/models/purchase_order.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrderReceivedByCategory(models.Model):
+    _name = "purchase.order.received.by.category"
+    _description = "Purchase Order Received by Category"
+
+    order_id = fields.Many2one("purchase.order")
+    category_id = fields.Many2one("product.category")
+    product_qty = fields.Float()
+    qty_received = fields.Float()
+    to_receive = fields.Float()
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    purchased_quantity_total = fields.Float(
+        string="Total Quantity", compute="_compute_purchased_quantity_total"
+    )
+    delivery_status = fields.Float(compute="_compute_delivery_status")
```

Comment:
Same remark about UoM

Reply:
if you need to be able to sum these two fields in pivot tables, consider storing the fields.

Reply:
Kind of overlaps with existing module [`purchase_reception_status`](https://github.com/OCA/purchase-workflow/tree/15.0/purchase_reception_status), althuogh that one is computing only a 'state', and not an actual percentage.

In any case, I think `delivery_status` should be renamed as `reception_percentage`, and if really needed, extracted out into a separate module that only does that